### PR TITLE
add missing dependencies for fslpy in easyconfig for FSL v6.0.5.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by branfosj — DISMISSED** [review]

File: easybuild/easyconfigs/f/FSL/FSL-6.0.5.1-foss-2021a.eb [hunk 1]
```diff
@@ -52,30 +52,40 @@ dependencies = [
...
     ('fslpy', '3.5.3', {
-        'installopts': '--prefix=%(installdir)s/fsl',
-        'source_urls': ['https://pypi.python.org/packages/source/f/fslpy/'],
         'checksums': ['f80ef8b7b49f475bfb71946619d33c118a7cbbe106c72f7777aaf14d9e261530'],
+        'installopts': '--prefix=%(installdir)s/fsl',
     }),
 ]
 
+modextrapaths = {
+    'PYTHONPATH': 'lib/python%(pyshortver)s/site-packages',
```
Comment (branfosj) [inline easybuild/easyconfigs/f/FSL/FSL-6.0.5.1-foss-2021a.eb:81]: See https://github.com/easybuilders/easybuild-easyconfigs/pull/19829/files#diff-120b4f2724977ead205a47a29d7e5aa11df4354f806000afd344d437ef4855f2R58

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3134
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3557.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.6.8
See https://gist.github.com/boegel/b50f247e2fba581110284ee31b7e3724 for a full test report.

File: easybuild/easyconfigs/f/FSL/FSL-6.0.5.1-foss-2021a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/f/FSL/FSL-6.0.5.1-foss-2021a.eb]: fixed in 6fe0d6bbfa

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3134
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3557.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.6.8
See https://gist.github.com/boegel/01fade7ea9aa85aed4ac34bf8b2f34cb for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3134
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/125910650cdb54f744802f54cb6e2698 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**